jitsi-utils/src/main/java/org/jitsi/utils/queue/QueueStatistics.java

```diff
@@ -88,54 +108,65 @@ public JSONObject getStats()
     {
         JSONObject stats = new JSONObject();
         long now = System.currentTimeMillis();
-        stats.put("added", totalPacketsAdded.get());
-        stats.put("removed", totalPacketsRemoved.get());
-        stats.put("dropped", totalPacketsDropped.get());
+        stats.put("added", totalPacketsAdded.sum());
+        stats.put("removed", totalPacketsRemoved.sum());
+        stats.put("dropped", totalPacketsDropped.sum());
         stats.put("add_rate", addRate.getRate(now));
         stats.put("remove_rate", removeRate.getRate(now));
         stats.put("drop_rate", dropRate.getRate(now));
         double duration = (now - firstPacketAddedMs) / 1000d;
         stats.put("duration_s", duration);
-        stats.put("average_remove_rate_pps", totalPacketsRemoved.get() / duration);
+        stats.put("average_remove_rate_pps", totalPacketsRemoved.sum() / duration);
```

Comment:
Add rate also seems like it'd be important here (so we can compare add rate and remove rate)

Reply:
Oops, I see that was already there.  What's the difference in units?  Maybe we should make those other stat names more obvious.  If they're not "items per second", what are they? Bps?

Reply:
I'm not actually 100% sure - these were all stats that were there previously.

That said, I believe the `getRate()` rates are calculated with `RateStatistics`, so over a 5 second window, whereas `average_remove_rate_pps` is calculated since the beginning of time.  The former is probably more useful...